### ISSUE TITLE
npm dependency failure on next for esprima-six

### ISSUE THREAD

**Issue by larkost** [rethinkdb/rethinkdb#5055]
On `next` we are getting a build failure in `npm` when fetching `browserify`. Looking manually at the `npm` registry `esprima-six` is no longer listed. My guess is that it is fairly old, since we are on version 3.24.13 of `browserify`, and the current is 12.0.1.

I suspect that we need to move up the version of `browerify` to something more recent, but am not sure why it is currently limited to `max_browserify_version=3.24.13` (in `./configure`).

@deontologician do you have any insight into why we are limiting this?

**Comment by AtnNn** [comment]
We added upper bounds on browserify and other node modules because some of them, mostly coffeescript, had breaking changes in minor versions.

**Comment by AtnNn** [comment]
It was discussed in https://github.com/rethinkdb/rethinkdb/issues/938 and https://github.com/rethinkdb/rethinkdb/issues/958

**Comment by deontologician** [comment]
We could bump this, it just needs to be tested that it works

**Comment by larkost** [comment]
I have a bump committed into `next` with c7fe5309. I changed the numbers in `configure`, `browserify.sh`, and `browserify.shrinkwrap`. Notably I did not change the lower-bound in `configure` or touch anything in `./admin/package.json`. This allows us to build, and some basic browsing both from my local build and one on `newton` (both with cleaned `./external` folders) work as expected.

I will be filing a separate bug to rationalize how we do all of this.
